ci-operator/config/openshift/installer/openshift-installer-release-4.16__okd-scos.yaml

```diff
@@ -140,7 +140,7 @@ resources:
       memory: 200Mi
 tests:
 - as: e2e-vsphere-ovn
-  optional: true
+  optional: false
```

Comment:
you need to add this test to the `openshift-installer-main__okd-scos.yaml` file. this is the config file for 4.16 which we no longer generate releases for